source/Octodiff.Tests/Octodiff.Tests.csproj

```diff
@@ -8,10 +8,10 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
-    <TargetFrameworks>net6.0;net462</TargetFrameworks>
```

Comment:
The library continues to single-target net6, so we should include net6.0 in TargetFrameworks here, not just 8.0

Reply:
The library actually targets `netstandard2.0`, I wasn't sure if we still needed .NET 6 here in this case.

Reply:
Ah... that's interesting.
If it targets netstandard, and we test it as working on net462, then if it also works on net8 that's probably good enough; yeah let's just go with net8 and net462 like you've done 👍